lib/src/mjpeg.dart

```diff
@@ -120,31 +136,35 @@ class _StreamManager {
   final bool isLive;
   final Duration _timeout;
   final Map<String, String> headers;
-  final Client _httpClient = Client();
+  final Client client;
```

Comment:
any reason to change the name and visibility here? I would prefer keeping it as it was if it's ok for you

Reply:
I changed the visibility because it's now a named parameter, they can't be private. Naming wise, no reaseon Ill change that

Reply:
It is not a named parameter that's why I asked :D

Reply:
Yes it is now, that's one of my major changes

Reply:
You change it for the widget, but not here :) check `_timeout` on the constructor just below. Params are not named for the manager

Reply:
I noticed, wasn't paying attention. haha

Reply:
Done 👍